### ISSUE TITLE
About training time

### ISSUE THREAD

**Issue by s0mnus112** [Yanfeng-Zhou/XNet#23]
Hello!
I wanted to ask about  your training time,I only ran 50 epochs in 12 hours while training the semi model.

**Comment by Yanfeng-Zhou** [comment]
Training time is related to many factors, such as GPU, dataset scale, model size, image size, etc. You need provide some detailed experimental settings so that I can judge whether this is reasonable.

**Comment by s0mnus112** [comment]
Thank you！
I used 4 RTX 3090（24GB），and i changed the batch_size to 8，It's still training slowly. I think maybe my dataset is too big,It's contains 24000  256*256 images.

**Comment by Yanfeng-Zhou** [comment]
You are right. For a dataset containing 24,000 256×256 images, it is a normal speed.I suggest reducing the image size during training, such as 256→128, reducing the total epochs, and increasing the batch size.

**Comment by s0mnus112** [comment]
Thank you so much!!🤞
